### ISSUE TITLE
_rev shouldn't be a key of an object

### ISSUE THREAD

**Issue by rickmcgeer** [LivelyKernel/lively.next#146]
In particular, one of the things we use objects for is dictionaries, and when the object keys can be anything, it's hard to tell a spurious entry just stuck in by the serializer.

**Comment by merryman** [comment]
Yes the serializer should remove these. Again, this needs to be fixed! Noted.
